CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## 23.11.0 (2023-11-30)
 
+### Special announcement
+
+New [`menuinst` v2](https://github.com/conda/menuinst/releases/tag/2.0.0) support!
+
+`conda` has supported Start menu items on Windows for a long time. This is what allows users to open up their Miniconda prompt on CMD with an initialized `conda` session. This menu item (or shortcut) creation logic is provided by the `menuinst` package.
```

Comment:
CMD is also called Command Prompt more specifically on Windows.

"... open up their Miniconda prompt on CMD (Command Prompt) with..."

Don't know if that's unnecessary for the audience we're addressing, though.

Reply:
Good suggestion!